sql/catalyst/src/main/java/org/apache/spark/sql/connector/util/V2ExpressionSQLBuilder.java

```diff
@@ -219,6 +218,10 @@ protected String inputToSQL(Expression input) {
     }
   }
 
+  protected String visitBinaryComparison(GeneralScalarExpression e) {
```

Comment:
this is not very different from overriding `def build` directly, as it's still a `GeneralScalarExpression`